app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ContactsController < ApplicationController
+  
+  def create
+    contact = Contact.new(contact_params)
+    contact.company_id = params[:company_id]
```

Comment:
same here! we talked in class about how to leverage your associations here rather than setting the foreign key manually. how could you refactor this?